### ISSUE TITLE
Is there pipeline for lncRNA?

### ISSUE THREAD

**Issue by hliu2016** [bcbio/bcbio-nextgen#2327]
Thanks a lot for your wonderful work.

lncRNA is already a hot topic in the research community. I wonder if there is a pipeline for lncRNA analysis in bcbio? if not, I think a function module embedded into the RNA-seq pipeline will be be very helpful.

**Comment by roryk** [comment]
Hi @hliu2016,

Thanks for the suggestion. You shouldn't have to do anything special for looking at lncRNA for organisms where it is annotated, it is quantified the same way as coding RNA is! Let us know if you had something special in mind that we aren't doing.

**Comment by hliu2016** [comment]
Hi @roryk
Thanks for your reply. For annotated organism, such as human and mouse, where can I find known lncRNAs and their quanitfied level from the output of bcbio?

In addition, for recogintion of lncRNAs, we often use the tools, including CPC, phyloCSF and CNCI, to evaluates their potential in coding proteins. Would you like to add process to do this?

**Comment by roryk** [comment]
Hi @hliu2016,

For pre-annotated lncRNA, they'll just be quantitated along with the rest of the genes in the supplied annotation. For assembly of novel isoforms/genes, we use CPAT http://rna-cpat.sourceforge.net/ to train a logistical model to predict coding/non-coding status of the new transcripts given the coding/non-coding status of the known transcripts.

**Comment by hliu2016** [comment]
Hi @roryk,
Thank you for your rapid reply. I noticed there is a `transcript_assembler` option to instruct bcbio carry out assembly of new gene and transcript. However, this option can only be set to cufflinks and/or sailfish, as described in bcbio manual. Then, CPAT is called defaultly by bcbio once transcript_assembler option is set, or it sould be explicitly called by setting corresponding parameter?

In addition, if I set the transcript_assembler to cufflinks or sailfish, then the alternative splicing events will be automatically called out for me? In fact, I am very interested in finding significant alternative splicing events.
